modules/oc-mirror-imageset-config-parameters-v2.adoc

```diff
@@ -0,0 +1,395 @@
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-imageset-config-parameters-v2_{context}"]
+= ImageSet configuration parameters for v2
+
+The oc-mirror plugin requires an image set configuration file that defines what images to mirror. The following table lists the available parameters for the `ImageSetConfiguration` resource.
+
+.`ImageSetConfiguration` parameters
+[cols="2,2a,1a",options="header"]
+|===
+|Parameter
+|Description
+|Values
+
+|`apiVersion`
+|The API version for the `ImageSetConfiguration` content.
+|String. For example: `mirror.openshift.io/v2alpha1`.
+
+|`archiveSize`
+|The maximum size, in GiB, of each archive file within the image set.
+|Integer. For example: `4`
+
+|`mirror`
+|The configuration of the image set.
+|Object
+
+|`mirror.additionalImages`
+|The additional images configuration of the image set.
+|Array of objects. For example:
+
+[source,yaml]
+----
+additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest
+----
+
+|`mirror.additionalImages.name`
+|The tag or digest of the image to mirror.
+|String. For example: `registry.redhat.io/ubi8/ubi:latest`
+
+|`mirror.blockedImages`
+|The full tag, digest, or pattern of images to block from mirroring.
+|Array of strings. For example: `docker.io/library/alpine`
+
+|`mirror.operators`
+|The Operators configuration of the image set.
+|Array of objects. For example:
+
+[source,yaml,subs="attributes+"]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
+    packages:
+      - name: elasticsearch-operator
+        minVersion: '2.4.0'
+----
+
+|`mirror.operators.catalog`
+|The Operator catalog to include in the image set.
+|String. For example: `registry.redhat.io/redhat/redhat-operator-index:v4.15`.
+
+|`mirror.operators.full`
+|When `true`, downloads the full catalog, Operator package, or Operator channel.
+|Boolean. The default value is `false`.
+
+|`mirror.operators.packages`
+|The Operator packages configuration.
+|Array of objects. For example:
+
+[source,yaml,subs="attributes+"]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
+    packages:
+      - name: elasticsearch-operator
+        minVersion: '5.2.3-31'
+----
+
+|`mirror.operators.packages.name`
+|The Operator package name to include in the image set
+|String. For example: `elasticsearch-operator`.
+
+|`mirror.operators.packages.channels`
+|The Operator package channel configuration.
+|Object
+
+|`mirror.operators.packages.channels.name`
+|The Operator channel name, unique within a package, to include in the image set.
+|String. For example: `fast` or `stable-v4.15`.
+
+|`mirror.operators.packages.channels.maxVersion`
+|The highest version of the Operator mirror across all channels in which it exists. 
+|String. For example: `5.2.3-31`
+
+|`mirror.operators.packages.channels.minVersion`
+|The lowest version of the Operator to mirror across all channels in which it exists. 
```

Comment:
i am not sure which description is correct, one has only `mirror` and other has `to mirror` and when user reads it meaning is completely changed, could you check this one too ?